azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/data/DbHelper.kt

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.data
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import android.provider.BaseColumns
+
+internal class DbHelper(context: Context) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
+    override fun onCreate(db: SQLiteDatabase) {
+        db.execSQL(CallHistoryContract.SQL_CREATE_CALL_HISTORY)
+    }
+
+    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
+    }
+
+    override fun onDowngrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
+    }
+
+    companion object {
+        // If you change the database schema, you must increment the database version.
+        const val DATABASE_VERSION = 1
+        const val DATABASE_NAME = "com.azure.android.communication.ui.calling.CallHistoryReader.db"
+    }
+}
+
+internal object CallHistoryContract {
+
+    const val ID = BaseColumns._ID
```

Comment:
I don't think you need a separate id field, especially once you simplify your cleanup logic to not address individual rows directly, and use a date range instead.

A sqlite db will already have a `rowid` unless you ask it not during table creation (see https://www.sqlite.org/rowidtable.html, esp the apology note at the bottom). So this id, roughly, becomes an alias for that field.

Reply:
I think it still makes sense to keep explicit id column due to:

The exception to this rule is when the rowid table declares an INTEGER PRIMARY KEY. In the exception, the INTEGER PRIMARY KEY becomes an alias for the rowid.

Therefore, applications should not normally access the rowid directly, but instead use an INTEGER PRIMARY KEY.

Reply:
As per offline meeting decided to keep ID